tests/integration/telemetry/outboundtrafficpolicy/helper.go

```diff
@@ -361,7 +361,7 @@ func setupEcho(t *testing.T, ctx resource.Context, mode TrafficPolicy) (echo.Ins
 			},
 			TLSSettings: &common.TLSSettings{
 				// Echo has these test certs baked into the docker image
-				RootCert:   mustReadCert(t, "cacert.pem"),
+				RootCert:   "",
```

Comment:
1. Just omit this field, the default value would be "".

2. Update the comment in TLSSettings.RootCert for the change that the system cert pool would be used if this is empty.